app/routes.py

```diff
@@ -123,8 +126,9 @@ def delete_election(link):
     for c in election.candidates.all():
         db.session.delete(c)
     
-    for v in election.votes.all():
-        db.session.delete(v)
+    if election.votes.all() is not None:
```

Comment:
I think this is Repeatitive. How about 
```suggestion
    votes = election.votes.all()
    if votes is not None:
        for v in votes:
        ...
```

So you won't have to make those calls twice

Reply:
it's basically the same thing but I will make the changes

Reply:
same result yes, but not the same thing. You shouldnt be making two database calls when you only need one.

Reply:
Database calls are expensive (memory and shii)

Reply:
True. I changed it shah

Reply:
True. I changed it shah